### ISSUE TITLE
Engines no longer work in series after this is installed.

### ISSUE THREAD

**Issue by Blue-Beaker** [JackRaidenPH/BuildCraftRF#5]
When this mod isn't present, engines can work in series with limited count. Stirling and Creative engines works in a series of 3, while Combustion engines works in a series of 5.
When using [Buildcraft Fluxified](https://www.curseforge.com/minecraft/mc-mods/buildcraft-fluxified) instead of this, the engines work as normal, but BC energy pipes works weird with RF generators instead.

**Comment by jackraidenph** [comment]
Wrong repo.

**Comment by Blue-Beaker** [comment]
No, the issue is about buildcraft rf:REfluxified, and it doesn't happen in buildcraft-fluxified. The later is just mentioned for reference because that one doesn't cause this problem.

**Comment by jackraidenph** [comment]
Oh, sorry then. The original text made an impression of actually being an issue of the beforementioned mod. I'll look at this then.

**Comment by DarianLStephens** [comment]
Was this ever looked in to?
I'm thinking of adding this because I want to use BC engines as a primary power source for my base.
